build.gradle.kts

```diff
@@ -263,3 +258,28 @@ fun Project.getOptionalExtra(key: String): String? {
         null
     }
 }
+
+fun Project.configureBintray(vararg publications: String) {
+    extensions.findByType<BintrayExtension>()?.run {
+        user = System.getenv("BINTRAY_USER")
+        key = System.getenv("BINTRAY_API_KEY")
```

Comment:
What if I don't have these envs? Will I get an understandable error?

Reply:
not sure, we should make and extension method for it